beta/kustomize/overlays/workspace/README.md

```diff
@@ -0,0 +1,20 @@
+# Workspace Overlay
+
+This section is only relevant if Google Workspace is your identity provider. 
+
+## Create a Google service account
+
+Create a Google service account and key as outlined in the 1Password support article [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client). 
+
+Download the credentials file provided by Google and save a copy to your 1Password account. 
+
+## Create a Kubernetes secret for the Google service account credential
```

Comment:
Perhaps edit the title to reference simply preparing to have the file in the correct location to have the secret created. In the main readme, you reference this for scimsession as "Prepare your `scimsession` file".